tests/session/session-getTransactionOptions-001.phpt

```diff
@@ -0,0 +1,92 @@
+--TEST--
+MongoDB\Driver\Session::getTransactionOptions()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto(); ?>
+<?php skip_if_server_version('<', '4.0'); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+$session = $manager->startSession();
+
+var_dump($session->getTransactionOptions());
+
+// Ending session until CDRIVER-3366 is fixed
+$session->endSession();
+
+$options = [
+    ['maxCommitTimeMS' => 0],
+    ['maxCommitTimeMS' => 1],
+    ['readConcern' => new \MongoDB\Driver\ReadConcern('majority')],
+    ['readPreference' => new \MongoDB\Driver\ReadPreference('primaryPreferred')],
+    ['writeConcern' => new \MongoDB\Driver\WriteConcern('majority')],
+];
+
+foreach ($options as $test) {
+    // Session no longer needs to be restarted once CDRIVER-3366 is fixed
+    $session = $manager->startSession();
+
+    $session->startTransaction($test);
+
+    var_dump($session->getTransactionOptions());
+
+    $session->abortTransaction();
```

Comment:
Along with `endSession()`, I'm not sure that this is needed since we're going to destroy the client session anyway when `$session` goes out of scope.